$NetBSD: patch-Makefile,v 1.2 2014/12/17 06:08:13 obache Exp $

* Fix man page installation path.
* `-a' option for cp(1) is not portable.

--- Makefile.orig	2013-05-16 02:28:56.000000000 +0000
+++ Makefile
@@ -35,10 +35,10 @@ test:
 PREFIX=/usr/local
 install: $(ALL)
 	mkdir -p "$(PREFIX)/bin"
-	mkdir -p "$(PREFIX)/share/man/man1"
+	mkdir -p "$(PREFIX)/${PKGMANDIR}/man1"
 	mkdir -p "$(PREFIX)/share/emacs/site-lisp"
 	mkdir -p "$(PREFIX)/share/doc/commit-patch"
-	cp -a $(BIN)   "$(PREFIX)/bin"
-	cp -a $(MAN)   "$(PREFIX)/share/man/man1"
-	cp -a $(ELISP) "$(PREFIX)/share/emacs/site-lisp"
-	cp -a $(DOC)   "$(PREFIX)/share/doc/commit-patch"
+	cp -RpP $(BIN)   "$(PREFIX)/bin"
+	cp -RpP $(MAN)   "$(PREFIX)/${PKGMANDIR}/man1"
+	cp -RpP $(ELISP) "$(PREFIX)/share/emacs/site-lisp"
+	cp -RpP $(DOC)   "$(PREFIX)/share/doc/commit-patch"
